humpbackdev/generator-humpback
# Fix setup xdebug mac and fix humpback

Closes #92 

# How to test:
- Using Mac
- Generate project
- Setup debugging as explained at https://humpback.readthedocs.io/en/latest/working-with-humpback/debugging-your-code-with-xdebug/
- Place a breakpoint in index.php; it should work
